### DISCUSSION TITLE
Welcome to openelb Discussions!

### DISCUSSION THREAD

**Discussion by FeynmanZhou** [openelb/openelb#228] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SurajKarmakar1** [comment]
Hello everyone, Im thinking of starting my first open source contribution here in openelb , can anyone guide me what are the things I should learn to get started ?
I know we need GOlang , kubernetes and Linux but how to get started any guidance will be helpful.

**Reply by renyunkang** [reply]
> Hello everyone, Im thinking of starting my first open source contribution here in openelb , can anyone guide me what are the things I should learn to get started ? I know we need GOlang , kubernetes and Linux but how to get started any guidance will be helpful.

The primary resource is the official documentation. You can start by learning from the Quick Start guide and then read through other relevant documents. Finally, you can explore the implementation details of OpenELB by examining the codebase.

**Reply by SurajKarmakar1** [reply]
Thanks alot , i will try my best to involve myself in the project and contribute some value.
